### PR TITLE
SALTO-5732: Fix Assets Component In Automation

### PR DESCRIPTION
**Pull request by idoamit8**
In this PR I fix assets component in Automation. 

---

_Additional context for reviewer_

---
_Release Notes_: 
_Jira Adpater:_
* assets component in Automation. 

---
_User Notifications_: 
None

### CODE REVIEW

**Review by shoshana-houri — APPROVED** [review]
🔥

File: packages/jira-adapter/src/filters/automation/automation_deployment.ts [hunk 1]
```diff
@@ -117,7 +117,7 @@ export type Component = {
 const ASSET_COMPONENT_SCHEME = Joi.object({
   value: Joi.object({
     schemaId: Joi.required(),
-  }).unknown(true),
+  }).unknown(true).required(),
```
Comment (shayc331) [inline packages/jira-adapter/src/filters/automation/automation_deployment.ts:120]: please add a unit test for this case

**Review by shayc331 — APPROVED** [review]
💣 
looking great, please add a unit test so it wont break again

File: packages/jira-adapter/src/filters/automation/automation_deployment.ts (hunk 1 above)
Comment (idoamit8) [reply packages/jira-adapter/src/filters/automation/automation_deployment.ts]: Done.

**Pull request merged**